.github/workflows/deployDev.yml

```diff
@@ -109,15 +110,3 @@ jobs:
         with:
           client-tarball: client.tgz
           deploy-env: ${{env.DEPLOY_ENV}}
-  verify-release:
-    runs-on: ubuntu-latest
-    needs: [deploy]
-    defaults:
-      run:
```

Comment:
Curious why we're no longer verifying the release for `dev`?

Reply:
This check is the first test of the cypress suite, which happens locally during the PR process and against the `test` environment once a merge goes through. This was only in place until we could run the whole suite against a remote environment in its entirety. 

I wouldn't mind this type of check for all of our remote environments, but it does take some time because of its dependence on spinning up cypress.

Reply:
I see, thanks for the explanation!